src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/ResolveSdksTaskTests.cs

```diff
@@ -188,6 +188,7 @@ public void UseLatestAndroidSdk (string buildtools, string jdk, ApiInfo[] apis,
 				UseLatestAndroidPlatformSdk = useLatestAndroidSdk,
 				AotAssemblies = false,
 				SequencePointsMode = "None",
+				AndroidApplication = true,
```

Comment:
Are these changes needed? given that the default for `ResolveAndroidTooling.AndroidApplication` is true?

Reply:
Technically not - but this matches what is happening in the .targets file where it always gets set. I made true default, just in case someone did what this test is doing manually so it wouldn't be breaking.